test/tests/util/awayKey.js

```diff
@@ -0,0 +1,28 @@
+/* @flow */
```

Comment:
I think this file should be `awaitKey.js`

Reply:
yep you're right. Please, I'll need extra 👀 on the test name as well...